### PR TITLE
[REF] models: Add 'create_date' value to __export__ xml_ids

### PR DESCRIPTION
**Pull request by moylop260**
This table could be huge and we need a way to identify old records in order to vacuum them

### CODE REVIEW

**Comment by moylop260** [comment]
@xmo-odoo @mart-e 
What do you think?

**Comment by xmo-odoo** [comment]
No objection, although using the helpers from `field.Datetime` would be cleaner.

**Comment by moylop260** [comment]
@xmo-odoo 
Thanks for your feedback!

I just used the helper `odoo.fields.Datetime.now()` as your recommended me

Regards!

**Comment by xmo-odoo** [comment]
@moylop260 I was going to merge this as-is, but looking a bit closer it looks like ir.model.data has the usual metadata fields (create_date/write_date) but also has its own versions thereof (date_init/date_update), and only the latter get filled by custom SQL (and not always either).

@mart-e apparently removed [some of these old fields last year](https://github.com/odoo/odoo/pull/34988) but didn't touch those of ir.model.data. So either we should finally remove those from ir.model.data as well, or *these* are the fields which should be manually updated here.

**Comment by mart-e** [comment]
Yes, I think we should remove the `date_init`, `date_update` fields (but this will need a bit of migration).
For this PR, you could simply update both :thinking:

File: odoo/models.py [hunk 1]
```diff
@@ -749,7 +749,7 @@ def to_xid(record_id):
             )))
             for r in missing
         )
-        fields = ['module', 'model', 'name', 'res_id']
+        fields = ['module', 'model', 'name', 'res_id', 'create_date']
```
Comment (luisg123v) [inline odoo/models.py:752]: Shouldn't the `write_date` field be set too?

Comment (xmo-odoo) [reply odoo/models.py:752]: @luisg123v #50661 subsumes this PR and adds an SQL-level default value to both `create_date` and `write_date` (on `ir.model.data`) so it handles both, and a few other improvements.

Comment (luisg123v) [reply odoo/models.py:752]: :+1:

**Comment by moylop260** [comment]
It will fixed from:
 - https://github.com/odoo/odoo/pull/50661

Thanks @xmo-odoo to take care about this.

**Pull request closed without merging**